src/index.html

```diff
@@ -2,11 +2,414 @@
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="nav-container">
+        <a href="#">
+          <img
+            src="./images/logo.png"
+            alt="MOYO logo of technic store"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__order">
+
+            <li class="nav__element">
+              <a
+                href="#"
+                class="is-active nav__element-link"
+              >
+              Apple
+              </a>
+            </li>
+
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+              >
+              Samsung
+              </a>
+            </li>
+
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+              >
+              Smartphones
+              </a>
+            </li>
+
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+                data-qa="nav-hover"
+              >
+              Laptosps & Computers
+              </a>
+            </li>
+
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+              >
+              Gadgets
+              </a>
+            </li>
+
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+              >
+              Tablets
+              </a>
+            </li>
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+              >
+              Photo
+              </a>
+            </li>
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+              >
+              Video
+              </a>
+            </li>
+
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="card-container">
+        <div
+        class="card"
+        data-qa="card"
+      >
+          <img
+          src="images/imac.jpeg"
+          alt="product item imac"
+          class="card__photo"
+        >
```

Comment:
wrong formatting for img, please fix it
` <img` has to be in the same line as `>`